### ISSUE TITLE
Lightning strikes on the inside of Wateridge

### ISSUE THREAD

**Issue by GoliathX211** [HolographicWings/LethalExpansion#35]
A teammate just died from getting struck by lightning while Wateridge was stormy from inside the dungeon.

**Comment by HolographicWings** [comment]
Yes i'm aware of this issue, i don't explain it, i increased the size of the sea for the next update, we'll see if that fixes the issue

**Comment by KawaiiJess** [comment]
@HolographicWings I just tested an r2modman profile with nothing besides BepInEx, LethalExpansion, and a few moons so at least one would be stormy, and the lightning is happening indoors on all moons. It notably only happens to equipment held by non-host players. I was just testing with 1 other friend, and her shovel was able to kill us indoors while mine was not. I have personally experienced this issue on Kast https://thunderstore.io/c/lethal-company/p/Ceelery/Kast/ and Taiga https://thunderstore.io/c/lethal-company/p/quackandcheese/MinecraftTaigaMoon/ while it sounds like @GoliathX211 had it happen on Wateridge.

Here's the r2modman profile code I was using: 018ce845-8c2c-e898-edde-fa5a2c0620f6

**Comment by HolographicWings** [comment]
> @HolographicWings I just tested an r2modman profile with nothing besides BepInEx, LethalExpansion, and a few moons so at least one would be stormy, and the lightning is happening indoors on all moons. It notably only happens to equipment held by non-host players. I was just testing with 1 other friend, and her shovel was able to kill us indoors while mine was not. I have personally experienced this issue on Kast https://thunderstore.io/c/lethal-company/p/Ceelery/Kast/ and Taiga https://thunderstore.io/c/lethal-company/p/quackandcheese/MinecraftTaigaMoon/ while it sounds like @GoliathX211 had it happen on Wateridge.
> 
> Here's the r2modman profile code I was using: 018ce845-8c2c-e898-edde-fa5a2c0620f6

Could you test on wateridge? I personally don't have the issue but if you have it I would like to know if it happen on wateridge especially

**Comment by 21Joakim** [comment]
This is caused by the same issue as https://github.com/HolographicWings/LethalExpansion/issues/20

**Comment by HolographicWings** [comment]
> This is caused by the same issue as https://github.com/HolographicWings/LethalExpansion/issues/20

Yes #85 gives me a clue about it

**Comment by KawaiiJess** [comment]
> Could you test on wateridge? I personally don't have the issue but if you have it I would like to know if it happen on wateridge especially

Is there a way to force stormy weather on a particular moon? Have you tried playing with a friend? As I stated, I was never able to make it happen on any moon with any configuration of mods as the host. It wasn't until I asked a friend to join me that it started happening again.

As an aside, I actually forgot I also saw #20 happen, and I believe I have video footage of it. I was the host and dead and the coil-head was completely ignoring my friend while I was spectating.
